projects/plugins/jetpack/require-lib.php

```diff
@@ -20,16 +16,29 @@ function jetpack_require_lib( $slug ) {
 	 * @param string $lib_dir Path to the library directory.
 	 */
 	$lib_dir = apply_filters( 'jetpack_require_lib_dir', $lib_dir );
-	$choices = array(
-		"$lib_dir/$slug.php",
-		"$lib_dir/$slug/0-load.php",
-		"$lib_dir/$slug/$basename.php",
-	);
-	foreach( $choices as $file_name ) {
-		if ( is_readable( $file_name ) ) {
-			require_once $file_name;
-			return;
-		}
+
+	$loaded_key = "{$lib_dir}{$slug}";
```

Comment:
Would've loved to use memoization on top, but the filter doesn't make that possible.